### PR TITLE
Add lnk to docs for getting started tutorial

### PR DESCRIPTION
**Pull request by clrcrl**
## Description & motivation
Hoping that adding the generated docs makes the tutorial a little more sensible.

Do you buy it?

## Pre-release docs
- [x] No: please ensure the base branch is `current`

### CODE REVIEW

File: website/docs/tutorial/1-setting-up.md [hunk 1]
```diff
@@ -3,16 +3,17 @@ title: Setting up
 id: setting-up
 ---
 
+In this tutorial, we will be turning the below query into a dbt project that is tested, documented, and deployed — you can check out the generated documentation for the project we're building [here](https://www.getdbt.com/getting-started-tutorial/#!/overview?g_v=1).
+
 This tutorial is geared at first-time users who want detailed instructions on how to go from zero to a deployed dbt project. You'll need a working knowledge of SQL in order to do this tutorial.
```
Comment (coapacetic) [inline website/docs/tutorial/1-setting-up.md:8]: Love the inclusion of this!

File: website/docs/tutorial/1-setting-up.md [hunk 2]
```diff
@@ -3,16 +3,17 @@ title: Setting up
...
 
 <LoomVideo id="cb99861ab1034f7fab5fa48529e61f85" />
 
 
 We recommend you go through this project once from beginning to end. Once you've completed it, you should go back through and read some of the FAQs to broaden your understanding of dbt.
 
-This tutorial is based on a fictional business named `jaffle_shop`, so you'll see this name used throughout the project.
+This tutorial is based on a fictional business named `jaffle_shop`, so you'll see this name used throughout the project. At this organization, the following query is used to build a `customers` table.
```
Comment (coapacetic) [inline website/docs/tutorial/1-setting-up.md:16]: This simple framing is huge for people to identify with how their organization is currently modeling and then seeing how dbt allows them to modularize (I think that's a word) their workflow

File: website/docs/learn/1-setting-up.md [hunk 3]
```diff
@@ -4,23 +4,24 @@ id: setting-up
 description: This tutorial is the required pre-work for dbt Learn.
 ---
 
+
 This tutorial is the required pre-work for dbt Learn. Before beginning this tutorial, please ensure that you been received your Snowflake credentials and dbt Cloud invitation by email from a Learn instructor.
 
-If you are **new to dbt**, we recommend you take some time to read through the FAQs, and come armed with any questions you have along the way. If you get completely stuck in this tutorial, please [email us](mailto:learn@getdbt.com) and we'll help unstick you.
+If you are **new to dbt**, we recommend you take some time to read through the FAQs, and come armed with any questions you have along the way. If you get completely stuck in this tutorial, please ask questions in the dedicated Slack channel, as detailed in the welcome email.
```
Comment (coapacetic) [inline website/docs/learn/1-setting-up.md:11]: Yes! please send questions to the slack - the sooner the better!

...where does learn@getdbt.com go anyways?

**Review by coapacetic — APPROVED** [review]
Subtle, yet powerful changes.  LGTM

File: website/docs/learn/1-setting-up.md (hunk 3 above)
Comment (clrcrl) [reply website/docs/learn/1-setting-up.md:11]: Does it not... go to you?

Comment (coapacetic) [reply website/docs/learn/1-setting-up.md:11]: I just sent an email to learn@getdbt.com - nothing coming through to kcoapman@fishtownanalytics.com*

**Review by jtcohen6 — COMMENTED** [review]
Sorry I was late in reviewing! This looks great, a definite improvement

**Pull request merged**